connectors/netty-connector/src/main/java/org/glassfish/jersey/netty/connector/internal/NettyInputStream.java

```diff
@@ -87,6 +87,8 @@ public int read(byte[] b, int off, int len) throws IOException {
 
```

Comment:
could you please update copyright year in both files (to 2019) - copyright header at the top of each file?

Reply:
thank you for changes, and a question - you are changing method ```read(byte[] b, int off, int len)``` but immediately under this method there is method ```read()``` without any parameters, but inner logic looks very similar, except the very reading. So, the question - shall  ```InputStream take``` be closed as well in the ```read()``` method?

Reply:
Indeed, I refactored the code a bit to avoid this mistake in the future.